src/components/UserCard.tsx

```diff
@@ -1,37 +1,51 @@
-import Image from 'next/image'
-
-// import sealife from '../../public/sealife-images/sealife_250px_001.webp'
+import JellyFishOutlineIcon from './icons/JellyFishOutlineIcon'
 
 type CardProps = {
 	name: string
-	storyPoints: number
+	storyPoints: string
 	imageSrc: string
+	showPoints: boolean
 }
 
-export default function UserCard({ name, storyPoints, imageSrc }: CardProps) {
+export default function UserCard({
+	name,
+	storyPoints,
+	imageSrc,
+	showPoints,
+}: CardProps) {
+	if (storyPoints === '-') showPoints = true
+
 	return (
-		<div className='card bg-base-100 w-28 h-28 shadow-xl rounded-lg border-2 border-gray-900 overflow-hidden'>
-			<figure className='z-0 w-28 h-28 object-cover relative block'>
-				<Image
+		<div className='card bg-base-100 size-32 shadow-xl rounded-lg border border-slate-600 overflow-hidden'>
+			<figure className='z-0 size-32 object-cover relative block'>
+				<img
 					src={imageSrc}
-					fill
-					alt='sealife image background'
-					style={{
-						objectFit: 'cover', // cover, contain, none
-					}}
+					alt='sealife background'
```

Comment:
suggestion: Imagine the page without styles, for a blind person. this alt alone doesn't make much sense. Instead you could say `{name} avatar profile` :)

Reply:
Good suggestion. I'll make this change now, but I was wanting to do a more comprehensive a11y discussion with you later since I'm still making and breaking many things. 
DONE